google-cloud-binary_authorization-v1/proto_docs/google/cloud/binaryauthorization/v1/service.rb

```diff
@@ -0,0 +1,189 @@
+        #     `ListAttestors` method to retrieve the next page of results.
+        class ListAttestorsResponse
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Request message for [BinauthzManagementService.DeleteAttestor][].
+        # @!attribute [rw] name
+        #   @return [::String]
+        #     Required. The name of the {::Google::Cloud::BinaryAuthorization::V1::Attestor attestors} to delete, in the format
+        #     `projects/*/attestors/*`.
+        class DeleteAttestorRequest
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Request to read the current system policy.
+        # @!attribute [rw] name
+        #   @return [::String]
+        #     Required. The resource name, in the format `locations/*/policy`.
+        #     Note that the system policy is not associated with a project.
+        class GetSystemPolicyRequest
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Request message for
+        # {::Google::Cloud::BinaryAuthorization::V1::ValidationHelperV1::Client#validate_attestation_occurrence ValidationHelperV1.ValidateAttestationOccurrence}.
+        # @!attribute [rw] attestor
+        #   @return [::String]
+        #     Required. The resource name of the {::Google::Cloud::BinaryAuthorization::V1::Attestor Attestor} of the
+        #     [occurrence][grafeas.v1.Occurrence], in the format
+        #     `projects/*/attestors/*`.
+        # @!attribute [rw] attestation
+        #   @return [::Grafeas::V1::AttestationOccurrence]
+        #     Required. An {::Grafeas::V1::AttestationOccurrence AttestationOccurrence} to
+        #     be checked that it can be verified by the Attestor. It does not have to be
+        #     an existing entity in Container Analysis. It must otherwise be a valid
+        #     AttestationOccurrence.
+        # @!attribute [rw] occurrence_note
+        #   @return [::String]
+        #     Required. The resource name of the [Note][grafeas.v1.Note] to which the
+        #     containing [Occurrence][grafeas.v1.Occurrence] is associated.
+        # @!attribute [rw] occurrence_resource_uri
+        #   @return [::String]
+        #     Required. The URI of the artifact (e.g. container image) that is the
+        #     subject of the containing [Occurrence][grafeas.v1.Occurrence].
+        class ValidateAttestationOccurrenceRequest
+          include ::Google::Protobuf::MessageExts
+          extend ::Google::Protobuf::MessageExts::ClassMethods
+        end
+
+        # Response message for
+        # {::Google::Cloud::BinaryAuthorization::V1::ValidationHelperV1::Client#validate_attestation_occurrence ValidationHelperV1.ValidateAttestationOccurrence}.
```

Comment:
`"ruby-cloud-service-override": "...ValidationHelperV1=ValidationHelper"` doesn't seem to be working in this output.